app/code/Magento/Cms/Test/Mftf/Page/CmsPageEditPage.xml

```diff
@@ -7,7 +7,7 @@
  -->
 <pages xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
        xsi:noNamespaceSchemaLocation="urn:magento:mftf:Page/etc/PageObject.xsd">
-    <page name="CmsPageEditPage" area="admin" url="admin/cms_page/edit/page_id/{{var}}" parameterized="true" module="Magento_Cms">
+    <page name="CmsPageEditPage" area="admin" url="cms/page/edit/page_id/{{var}}" parameterized="true" module="Magento_Cms">
```

Comment:
Could you, by the way, change this generic `{{var}}` to something more meaningful?
```suggestion
    <page name="CmsPageEditPage" area="admin" url="cms/page/edit/page_id/{{pageId}}" parameterized="true" module="Magento_Cms">
```